### PR TITLE
Feature/measure before tool

### PR DESCRIPTION
**Pull request by janphilipp-winkler**

### CODE REVIEW

File: components/ToolMoodMeasurer/ToolMoodMeasurer.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { useState } from "react";
+import Slider from "../Slider/Slider";
+import NavButton from "../NavButton/NavButton";
+import Guide from "../Guide/Guide";
+import * as Styled from "./ToolMoodMeasurer.styled";
+
+export default function ToolMoodMeasurer({ page, setPage }) {
+  const [before, setBefore] = useState(0.5);
+  const [after, setAfter] = useState(0.5);
```
Comment (JessicaLoers) [inline components/ToolMoodMeasurer/ToolMoodMeasurer.js:9]: **Just an egghead 🥚 suggestion**
_You can use derived state in your component to simplify the state management._

_Not tested example_

```js
const [mood, setMood] = useState({ before: 0.5, after: 0.5 });

function handleSliderChange(event) {
  setMood((prevMood) => ({
    ...prevMood,
    [page === 0 ? 'before' : 'after']: event.target.value,
  }));
}
```

**Review by JessicaLoers — CHANGES_REQUESTED** [review]
**Hey Mood Tracker Team. ✌️**

Your code is quite neat and follows a good structure. However, there are always ways to enhance readability, maintainability, and efficiency. 

_Please see my comments and suggestions for improvements. 💙_

File: components/ToolMoodMeasurer/ToolMoodMeasurer.js (hunk 1 above)
Comment (janphilipp-winkler) [reply components/ToolMoodMeasurer/ToolMoodMeasurer.js:9]: thank you for the idea but ... no :D (can't wrap my 🥚-head around it...)

**Comment by janphilipp-winkler** [comment]
> **Hey Mood Tracker Team. ✌️**
> 
> Your code is quite neat and follows a good structure. However, there are always ways to enhance readability, maintainability, and efficiency.
> 
> _Please see my comments and suggestions for improvements. 💙_

Thanks a lot Jessica for the thorough review of my code :)

**Pull request merged**